fix(sap jco3): addition of sapjco3.jar lib details (#729)
bonitasoft/bonita-doc: md/sap-jco-3.md

Contents of `sapjco-ntamd64-3.0.3.zip`

* `Readme.txt`: contains instructions

> * `sapjco3.jar`: must be installed in the `/endorsed` directory of your Bonita Studio installation and in class path of the bonita webapp.

Reviewer: ```suggestion
* `sapjco3.jar`: must be installed in the `/endorsed` directory of your Bonita Studio installation and in class path of the bonita web server.
```

> * `sapjcorfc.dll` (`sapjcorfc.so`): must be installed in the native library search path:

Reviewer: ```suggestion
* `sapjco3.dll` (`libspajco3.so`): must be installed in the native library search path:
```

  * Windows: usually the dll file is stored in `C:\windows\system32`
  * Linux: usually the dll file is stored in `/usr/lib`
* `javadoc`: contains the .html help pages for installation
* `examples`: contains some examples

### How to use the contents of the sapjco-ntamd64-3.0.3.zip file with an application server?

#### 1. Install the system library.

##### Windows

1. Store the `sapjco.dll` file in the `C:\windows\system32` directory.

> 2. Reboott

Reviewer: ```suggestion
2. Reboot
```

##### Linux

> Store the `sapjco.so` in the `/usr/lib` directory.

Reviewer: ```suggestion
Store the `libsapjco3.so` in the `/usr/lib` directory.
```

#### 2. Configure the application server

##### Tomcat example (BonitaSubscription-7.8.3-Tomcat-8.5.34 bundle)

It is assumed that both the Tomcat and the Bonita Engine were already successfully started once.

1. Stop Tomcat

> 2. Copy the `sapjco3.jar` file into `BonitaSubscription-7.8.3-Tomcat-8.5.34\server\webapps\bonita\WEB-INF\lib` directory

Reviewer: ```suggestion
2. Copy the `sapjco3.jar` file into `BonitaSubscription-7.8.3-Tomcat-8.5.34\server\lib` directory
```
